### PR TITLE
Puts the Fist of Sol back in the bag.

### PR DESCRIPTION
**Pull request by QAI-07**
## About The Pull Request

For a while now, on rare occasions, the Fist of Sol would appear in the Rubicon sector along with an identical clone of itself. While it still works as intended, it's appearance outside of strict admin intervention was _not_ intended. I've just finally gotten up to fix it.

## Why It's Good For The Game

Fix man good.
Also, I can now buff it without worry of it appearing randomly and jumpscaring the crew with funky new weapons.

## Changelog
:cl:
fix: fixed the Fist of Sol Boss showing up when it shouldn't. 
/:cl:

### CODE REVIEW

File: nsv13/code/modules/overmap/ai-skynet.dm [hunk 1]
```diff
@@ -894,9 +894,9 @@ Adding tasks is easy! Just define a datum for it.
 	faction = FACTION_ID_SYNDICATE
 	size = 1
 	allow_difficulty_scaling = FALSE
-	battleship_types = list(/obj/structure/overmap/syndicate/ai/fistofsol)
+	battleship_types = list(/obj/structure/overmap/hostile/ai/fistofsol)
 	supply_types = list(/obj/structure/overmap/syndicate/ai/carrier/elite)
-	taunts = list("That's it... Just you and me now, no support, no distractions... no war. Whoever wins is the best crew.")
+	taunts = list("What a pleasure that we should meet again. I hope you won't disapoint!")
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/overmap/ai-skynet.dm:899]:
```suggestion
	taunts = list("What a pleasure that we should meet again. I hope you won't disappoint!")
```

**Review by Bokkiewokkie — COMMENTED** [review]
just a small thing...

**Review by DeltaFire15 — DISMISSED** [review]
Instead of changing its type away from ../syndicate/ai which leads to issues with things that check for this subtype, add it to the enemy_blacklist in ../subsystem/starsystem.dm.

**Comment by red031000** [comment]
image
doesn't work

**Comment by red031000** [comment]
ok, correction, does work, however the same issue happens to sol's revenge, might be an idea to fix it as well?

**Comment by Pontenerd** [comment]
image
round id is 4800

**Comment by Bokkiewokkie** [comment]
> image round id is 4800

SyndicateFlagshipInfograph

**Comment by Bokkiewokkie** [comment]
@QAI-07 Are you going to adress DeltaFire's requested changes?

**Comment by QAI-07** [comment]
@Bokkiewokkie I would if I had even the slightest idea of how to do any of that.

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

**Review by Bokkiewokkie — APPROVED** [review]

**Pull request merged**